app/main.py

```diff
@@ -42,7 +42,7 @@ def create_tables(engine, psql_environment):
 set_ui_language()
 
 from app.routers import (  # noqa: E402
-    agenda, calendar, categories, celebrity, currency, dayview,
+    agenda, calendar, categories, celebrity, currency, credits, dayview,
```

Comment:
Reorder - credits should be before currency

Reply:
fixed all the notes, ready for merge :)